src/ui/React/LogBoxManager.tsx

```diff
@@ -249,7 +249,12 @@ function LogWindow(props: IProps): React.ReactElement {
 
   function title(full = false): string {
     const maxLength = 30;
-    const t = `${script.filename} ${script.args.map((x: ScriptArg): string => `${x}`).join(" ")}`;
+    const server = GetServer(script.server);
```

Comment:
script.server here is already a hostname, is there a reason you're doing another lookup to fetch the actual server using that name?